insights/parsers/vdsm_conf.py

```diff
@@ -1,38 +1,92 @@
 """
 VDSMConfIni - file ``/etc/vdsm/vdsm.conf``
-==========================================
+------------------------------------------
+VDSMLoggerConf - file ``/etc/vdsm/logger.conf``
+-----------------------------------------------
+"""
 
-The VDSM configuration file ``/etc/vdsm/vdsm.conf`` is in the standard 'ini'
-format and is read by the IniConfigFile parser.
+from .. import IniConfigFile, parser
+from insights.specs import Specs
 
-Sample configuration file::
 
-    [vars]
-    ssl = true
-    cpu_affinity = 1
+@parser(Specs.vdsm_conf)
+class VDSMConfIni(IniConfigFile):
+    """``VDSMConfIni`` parsers configuration file ``/etc/vdsm/vdsm.conf``
+    format and is read by the IniConfigFile parser.
```

Comment:
@psachin it looks like you left out text in the middle of this description, `is in the standard 'ini'`.  You might consider rewording this just a bit to make it read clearer.